src/core/renderController.ts

```diff
@@ -24,26 +24,28 @@ module Plottable {
     var _componentsNeedingComputeLayout = new Utils.Set<Component>();
     var _animationRequested = false;
     var _isCurrentlyFlushing = false;
+    export module Policy {
+      export var IMMEDIATE = "immediate";
+      export var ANIMATION_FRAME = "animationframe";
+      export var TIMEOUT = "timeout";
+    }
     export var _renderPolicy: RenderPolicies.RenderPolicy = new RenderPolicies.AnimationFrame();
 
-    export function setRenderPolicy(policy: string | RenderPolicies.RenderPolicy): void {
-      if (typeof(policy) === "string") {
-        switch ((<string> policy).toLowerCase()) {
-          case "immediate":
-          policy = new RenderPolicies.Immediate();
-          break;
-          case "animationframe":
-          policy = new RenderPolicies.AnimationFrame();
-          break;
-          case "timeout":
-          policy = new RenderPolicies.Timeout();
-          break;
-          default:
-          Utils.Methods.warn("Unrecognized renderPolicy: " + policy);
-          return;
-        }
+    export function setRenderPolicy(policy: string): void {
+      switch ((<string> policy).toLowerCase()) {
+        case Policy.IMMEDIATE:
+        _renderPolicy = new RenderPolicies.Immediate();
+        break;
+        case Policy.ANIMATION_FRAME:
+        _renderPolicy = new RenderPolicies.AnimationFrame();
+        break;
+        case Policy.TIMEOUT:
+        _renderPolicy = new RenderPolicies.Timeout();
+        break;
+        default:
+        Utils.Methods.warn("Unrecognized renderPolicy: " + policy);
+        return;
```

Comment:
we don't have any tslint rule to enforce this, but try to stick to some consistency. In general the statements after the `case`, including the `break` are on indented line